kubejs/server_scripts/enigmatica/kubejs/normal/recipetypes/mythicbotany/infusion.js

```diff
@@ -2,6 +2,9 @@ onEvent('recipes', (event) => {
     if (global.isNormalMode == false) {
         return;
     }
+
+    // todo: id_prefix
```

Comment:
remember todo :D

Reply:
that todo isn't for me, i left it for you guys, quickly glanced across that file and notice it didn't use prefixes ^-^